scenes/tileset.tres

```diff
@@ -153,6 +153,7 @@ texture_region_size = Vector2i(64, 64)
 2:2/0 = 0
 2:2/0/terrain_set = 0
 2:2/0/terrain = 0
+2:2/0/physics_layer_0/polygon_0/points = PackedVector2Array(-32, -32, -32, 32, 32, 32, 32, -32)
```

Comment:
You should revert this change to add a collision to the grass:

<img width="507" height="439" alt="Image" src="https://github.com/user-attachments/assets/22426190-28d8-4ddf-ab66-8a51ee021518" />

This change breaks, for example, this corner in Fray's End:

<img width="640" height="360" alt="Image" src="https://github.com/user-attachments/assets/f3918d44-baf5-4912-9299-e0de31476cf6" />